modules/ztp-configuring-ipsec-using-ztp-and-siteconfig.adoc

```diff
@@ -44,23 +41,21 @@ interfaces:
 - name: hosta_conn
   type: ipsec
   libreswan:
-    left: <cluster_node> <1>
+    left: '%defaultroute'
     leftid: '%fromcert'
     leftmodecfgclient: false
-    leftcert: <left_cert> <2>
+    leftcert: left_server
     leftrsasigkey: '%cert'
-    right: <external_host> <3>
+    right: <external_host> <1>
     rightid: '%fromcert'
     rightrsasigkey: '%cert'
-    rightsubnet: <external_address> <4>
-    ikev2: insist <5>
+    rightsubnet: <external_address> <2>
+    ikev2: insist <3>
     type: tunnel
 ----
-<1> Replace `<cluster_node>` with the IP address or DNS hostname of the cluster node for the cluster-side IPsec tunnel.
-<2> Replace `<left_cert>` with the IPsec certificate nickname.
-<3> Replace `<external_host>` with the external host IP address or DNS hostname.
-<4> Replace `<external_address>` with the IP address or subnet of the external host on the other side of the IPsec tunnel.
-<5> Use the IKEv2 VPN encryption protocol only. Do not use IKEv1, which is deprecated.
+<1> Replace `<external_host>` with the external host IP address or DNS hostname.
+<2> Replace `<external_address>` with the IP address or subnet of the external host on the other side of the IPsec tunnel.
```

Comment:
Same comment as before about the subnet.

Reply:
Updated as per the suggestion.